aten/src/ATen/core/type_conversion.h

```diff
@@ -0,0 +1,217 @@
+#pragma once
```

Comment:
Can you stack the move to this file under this PR? It's hard for me to tell what is a mechanical move and what is added functionality.